AGENTS.md

```
@@ -0,0 +1,253 @@
+5. **Performance**: Keep components lightweight, lazy-load when possible
+
+## Git Workflow
+
+### Branch Strategy
+- `main`: Production-ready code
+- Feature branches: `feature/description` or `fix/description`
+- Create PR for review before merging
+
+### Commit Messages
+Follow conventional commits:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)